.ci/build.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -xueo pipefail
 
-# TODO: make sdist work on all, it currently fails for clash-cosim
-cabal v2-sdist clash-prelude clash-lib clash-ghc
```

Comment:
I'd keep the TODO and add a note that this only works after building clash-cosim.